main.tex

```diff
@@ -172,8 +172,8 @@ \section{Methods: Recognising Extortion}\label{sec:sserror-zd-strategies}
     \tilde q=\alpha S_x + \beta S_y + \gamma
 \end{equation}
 
- where \(\tilde p = (1 - p_1, 1 - p_2, p_3, p_4)\) and
-\(\tilde q = (1 - q_1, 1 - q_2, q_3, q_4)\) then:
+where \(\tilde p = (p_1 - 1, p_2 - 1, p_3, p_4)\) and
```

Comment:
According to [1], \tilde q = q_1 - 1, q3, q2 - 1, q4

[1] https://www.pnas.org/content/109/26/10409